database/server.go

```diff
@@ -197,12 +197,15 @@ func execSelect(c redis.Connection, mdb *Server, args [][]byte) redis.Reply {
 	return protocol.MakeOkReply()
 }
 
-func (server *Server) flushDB(dbIndex int) redis.Reply {
+func (server *Server) flushDB(dbIndex int, isFlushAll bool) redis.Reply {
```

Comment:
The parameter `isFulshAll` is very confusing. We could create two functions，a internal function to flush db only invoked by flushAll, and another function invoked by Exec which has AOF and all necessary procedures.

Reply:
I was wrong. I thought about it this way. Before that, I thought adding another function was a bad behavior.